Websites/perf.webkit.org/public/v3/models/build-request.js

```diff
@@ -121,7 +121,7 @@ class BuildRequest extends DataModelObject {
                     continue;
                 if (buildRequest.hasCompleted())
                     return buildRequest;
-                if (buildRequest.isScheduled()
+                if ((buildRequest.isScheduled() || requestIdsWillBeUpdatedToScheduled.includes(parseInt(buildRequest.id())))
```

Comment:
`+buildRequest.id()` would be better in that it would require the whole string needs to be digits. e.g. 123abc will result in NaN instead of 123 with parseInt.

Reply:
Done